src/core/connection.c

```diff
@@ -2443,6 +2452,7 @@ QuicConnHandshakeConfigure(
 
     Connection->State.Started = TRUE;
     Connection->Stats.Timing.Start = QuicTimeUs64();
+    QuicPerfCounterAdd(ConnectionsActive, 1);
```

Comment:
I think it's better to increment this at the same point of create and decrement on free.

Reply:
But won't that count connections that never do anything?

Reply:
It's more consistent to do it the way proposed by Nick.